collect-config-setup/fragments/configure_config_agent.sh

```diff
@@ -74,8 +74,7 @@ mkdir -p $oac_templates/var/run/heat-config
 echo "{{deployments}}" > $oac_templates/var/run/heat-config/heat-config
 
 # os-refresh-config scripts directory
-# This moves to /usr/libexec/os-refresh-config in later releases
-orc_scripts=/opt/stack/os-config-refresh
+orc_scripts=/usr/libexec/os-refresh-config
```

Comment:
No, I was already getting deprecation messages when using the old value, indicating that the new one is correct. I also tested with the new value on OSP8 successfully.